internal/dispatch/graph/computecheck_test.go

```diff
@@ -104,7 +103,7 @@ func TestComputeCheckWithCaveats(t *testing.T) {
 				{
 					"document:foo#view@user:sarah",
 					map[string]any{
-						"somecondition": 42,
+						"somecondition": "42",
```

Comment:
For testing: we need to test that string conversion to int also works, so I figured this would be a good way to do so as part of a "real world" like usage

Reply:
In practice, since the input is either a numbervalue (which is always a float64) or a stringvalue, it should always Just Work (TM)